DrawingEngine/DrawingEngine.h

```diff
@@ -0,0 +1,25 @@
+#ifndef DRAWINGENGINE_H
+#define DRAWINGENGINE_H
+
+#include <iostream>
+#include <memory>
+#include <any>
+#include <vector>
+#include "../Logs/Logs.h"
+#include "../ClassFactory/ClassFactory.h"
+
+using namespace std;
```

Comment:
https://isocpp.github.io/CppCoreGuidelines/CppCoreGuidelines#sf7-dont-write-using-namespace-at-global-scope-in-a-header-file